### PR TITLE
Ensure header doesn't take less space if information is omitted

### PR DESCRIPTION
**Pull request by SnowyMouse**
From earlier, I made a pull request (#2445) to fix the alignment for mail for the action buttons jumping around. I was sort of able to fix contacts, too, but an issue was found that caused it to jump around if either the main or secondary line is omitted. Here's a pull request that fixes that issue while also having the multi contact view account for this, too.

This isn't perfect. As stated previously, long names can result in multiple lines being taken, and there's no way to prevent this outside of maybe truncation. However, I can make it so if a line isn't present, it doesn't affect the spacing. This can happen if you either don't fill out any of the name fields or fill out any fields that go into the second line such as birthdays.

I personally find this to improve the appearance, since it not only fixes the contact buttons but also makes it so these lines appear at the same place (provided one of the lines isn't so long that it wraps to a second line)

test
.

### CODE REVIEW

File: src/contacts/ContactViewer.js [hunk 1]
```diff
@@ -113,14 +113,18 @@ export class ContactViewer {
 					m(".header.pt-ml", [
 						m(".contact-actions.flex-space-between.flex-wrap.mt-xs", [
 							m(".left.flex-grow-shrink-150", [
-								m(".h2.selectable.text-break", this.contactAppellation),
-								m(".flex-wrap.selectable", insertBetween([
+								m(".h2.selectable.text-break", [
+									this.contactAppellation,
+									m.trust("&nbsp;") // alignment in case nothing is present here
```
Comment (charlag) [inline src/contacts/ContactViewer.js:118]: We try to track and check each `m.trust()` usage, I think `\u00A0` should work as well?

File: src/contacts/MultiContactViewer.js [hunk 2]
```diff
@@ -28,11 +28,14 @@ export class MultiContactViewer {
 				m(".fill-absolute.mt-xs.plr-l",
 					(contactView._contactList && contactView._contactList.list.getSelectedEntities().length > 0) ? [
 							// Add spacing so buttons for contacts also align with the regular client view's buttons
-							m(".header.pt-ml",
-								m(".contact-actions.flex-space-between.flex-wrap", [
-									m(".flex.items-center", this._getContactSelectionMessage(contactView)),
-									m(actionBar)
-								])
+							m(".header.pt-ml.flex-space-between",
+								m(".left.flex-grow-shrink-150", [
```
Comment (charlag) [inline src/contacts/MultiContactViewer.js:32]: I don't see why it should be `.flex-grow-shrink-150`, it can be just `.flex-grow`

**Review by charlag — COMMENTED** [review]
Hi!
Thanks for the PR, it works very well for me and feels much better.
There are just tiny suggestions I have

File: src/contacts/ContactViewer.js (hunk 1 above)
Comment (SnowyMouse) [reply src/contacts/ContactViewer.js:118]: It does! I'll use that instead.

File: src/contacts/MultiContactViewer.js (hunk 2 above)
Comment (SnowyMouse) [reply src/contacts/MultiContactViewer.js]: Got it!

**Comment by SnowyMouse** [comment]
All right. I've read your suggestions and pushed a couple of commits.

**Review by charlag — COMMENTED** [review]
Thank you for changes and sorry for the delay!
May I have another small ask? Can we pull "u00A0" into a constant somewhere in Utils or so, maybe named NBSP?

**Comment by SnowyMouse** [comment]
Sure thing! All done.

**Comment by charlag** [comment]
Thanks!

**Pull request merged**